### ISSUE TITLE
`android-jni-wrappers` error "use of undeclared identifier 'assert'"

### ISSUE THREAD

**Issue by CraigHutchinson** [KhronosGroup/OpenXR-SDK-Source#275]
The current master did not compile for Android cross-compiling as there is a missing `#include <cassert>` in the `android-jni-wrappers` code.

The full compiler error is as follows but occurs in multiple places:

```
C:\Projects\HelloXR\out\deps_src\openxr\src\external\android-jni-wrappers\wrap\ObjectWrapperBase.h(262,5): error G30564CF7: use of undeclared identifier 'assert'
      assert(!obj.isNull());
      ^
```

I'm guessing this is 'external' code by the path so maybe needs propagating up stream?

**Comment by rpavlik** [comment]
Ah, interesting, good catch. Wonder why I didn't see it here. The upstream is also by me, https://gitlab.freedesktop.org/monado/utilities/android-jni-wrappers

**Comment by rpavlik** [comment]
I am curious what cross-compile environment you used that triggered this issue, you can see the CI ones I used that didn't. But in any case, thanks for the report, I've fixed it in #280, please confirm it works for you!

**Comment by androidzhibinw** [comment]
Hello @rpavlik looks like it is related with NDK version, when use NDK 23 , I can see a  similar assert error, but no error for earlier NDK version.
